podcasts/Notifications/NotificationsCoordinator.swift

```diff
@@ -234,18 +234,62 @@ enum NotificationsGroup: CaseIterable {
         }
     }
 
+    func trigger(order: Int, notification: NotificationType) -> UNNotificationTrigger? {
```

Comment:
I'm not 100% sure on the dates returned here and I wonder if we might add some tests just to verify.